--- cassandra-cassandra-3.5/src/java/org/apache/cassandra/thrift/CustomTThreadPoolServer.java	2016-04-09 06:22:00.000000000 +0200
+++ src/java/org/apache/cassandra/thrift/CustomTThreadPoolServer.java	2016-05-31 15:19:09.777021914 +0200
@@ -188,9 +188,9 @@
             TProtocol inputProtocol = null;
             TProtocol outputProtocol = null;
             SocketAddress socket = null;
-            try (TTransport inputTransport = inputTransportFactory_.getTransport(client_);
-                 TTransport outputTransport = outputTransportFactory_.getTransport(client_))
-            {
+	    TTransport inputTransport = inputTransportFactory_.getTransport(client_);
+            TTransport outputTransport = outputTransportFactory_.getTransport(client_);
+            try {
                 socket = ((TCustomSocket) client_).getSocket().getRemoteSocketAddress();
                 ThriftSessionManager.instance.setCurrentSocket(socket);
                 processor = processorFactory_.getProcessor(client_);
@@ -226,7 +226,10 @@
             {
                 if (socket != null)
                     ThriftSessionManager.instance.connectionComplete(socket);
-
+		if (inputTransport != null)
+           	    inputTransport.close();
+		if (outputTransport != null)
+           	    outputTransport.close();
                 activeClients.decrementAndGet();
             }
         }
