--- cassandra-cassandra-3.5/test/unit/org/apache/cassandra/utils/NoSpamLoggerTest.java	2016-04-09 06:22:00.000000000 +0200
+++ test/unit/org/apache/cassandra/utils/NoSpamLoggerTest.java	2016-05-31 16:39:22.118191738 +0200
@@ -39,7 +39,7 @@
 {
     Map<Level, Queue<Pair<String, Object[]>>> logged = new HashMap<>();
 
-   Logger mock = new SubstituteLogger(null)
+   Logger mock = new SubstituteLogger(null, null, true)
    {
 
        @Override
