diff -ur cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/analyzer/StandardAnalyzer.java cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/analyzer/StandardAnalyzer.java
--- cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/analyzer/StandardAnalyzer.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/analyzer/StandardAnalyzer.java	2016-10-17 13:43:49.832944178 +0200
@@ -33,7 +33,7 @@
 import com.google.common.annotations.VisibleForTesting;
 
 import com.carrotsearch.hppc.IntObjectMap;
-import com.carrotsearch.hppc.IntObjectOpenHashMap;
+import com.carrotsearch.hppc.IntObjectHashMap;
 
 public class StandardAnalyzer extends AbstractAnalyzer
 {
@@ -48,7 +48,7 @@
         KATAKANA(12),
         HANGUL(13);
 
-        private static final IntObjectMap<TokenType> TOKENS = new IntObjectOpenHashMap<>();
+        private static final IntObjectMap<TokenType> TOKENS = new IntObjectHashMap<>();
 
         static
         {
diff -ur cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/disk/DynamicTokenTreeBuilder.java cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/disk/DynamicTokenTreeBuilder.java
--- cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/disk/DynamicTokenTreeBuilder.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/disk/DynamicTokenTreeBuilder.java	2016-10-17 13:43:49.833944186 +0200
@@ -23,7 +23,7 @@
 import org.apache.cassandra.utils.AbstractIterator;
 import org.apache.cassandra.utils.Pair;
 
-import com.carrotsearch.hppc.LongOpenHashSet;
+import com.carrotsearch.hppc.LongHashSet;
 import com.carrotsearch.hppc.LongSet;
 import com.carrotsearch.hppc.cursors.LongCursor;
 
@@ -49,7 +49,7 @@
     {
         LongSet found = tokens.get(token);
         if (found == null)
-            tokens.put(token, (found = new LongOpenHashSet(2)));
+            tokens.put(token, (found = new LongHashSet(2)));
 
         found.add(keyPosition);
     }
@@ -70,7 +70,7 @@
         {
             LongSet found = tokens.get(newEntry.getKey());
             if (found == null)
-                tokens.put(newEntry.getKey(), (found = new LongOpenHashSet(4)));
+                tokens.put(newEntry.getKey(), (found = new LongHashSet(4)));
 
             for (LongCursor offset : newEntry.getValue())
                 found.add(offset.value);
diff -ur cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/disk/OnDiskIndexBuilder.java cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/disk/OnDiskIndexBuilder.java
--- cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/disk/OnDiskIndexBuilder.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/disk/OnDiskIndexBuilder.java	2016-10-17 13:43:49.833944186 +0200
@@ -189,7 +189,7 @@
         minKey = (minKey == null || keyComparator.compare(minKey, key.getKey()) > 0) ? key.getKey() : minKey;
         maxKey = (maxKey == null || keyComparator.compare(maxKey, key.getKey()) < 0) ? key.getKey() : maxKey;
 
-        // 60 ((boolean(1)*4) + (long(8)*4) + 24) bytes for the LongOpenHashSet created when the keyPosition was added
+        // 60 ((boolean(1)*4) + (long(8)*4) + 24) bytes for the LongHashSet created when the keyPosition was added
         // + 40 bytes for the TreeMap.Entry + 8 bytes for the token (key).
         // in the case of hash collision for the token we may overestimate but this is extremely rare
         estimatedBytes += 60 + 40 + 8;
diff -ur cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/disk/TokenTree.java cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/disk/TokenTree.java
--- cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/disk/TokenTree.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/disk/TokenTree.java	2016-10-17 13:43:49.834944194 +0200
@@ -27,7 +27,7 @@
 import org.apache.cassandra.index.sasi.utils.RangeIterator;
 import org.apache.cassandra.utils.MergeIterator;
 
-import com.carrotsearch.hppc.LongOpenHashSet;
+import com.carrotsearch.hppc.LongHashSet;
 import com.carrotsearch.hppc.LongSet;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Function;
@@ -410,7 +410,7 @@
 
         public LongSet getOffsets()
         {
-            LongSet offsets = new LongOpenHashSet(4);
+            LongSet offsets = new LongHashSet(4);
             for (TokenInfo i : info)
             {
                 for (long offset : i.fetchOffsets())
@@ -520,4 +520,4 @@
             return index < offsets.length ? keyFetcher.apply(offsets[index++]) : endOfData();
         }
     }
-}
\ No newline at end of file
+}
diff -ur cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/memory/KeyRangeIterator.java cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/memory/KeyRangeIterator.java
--- cassandra-cassandra-3.9/src/java/org/apache/cassandra/index/sasi/memory/KeyRangeIterator.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/src/java/org/apache/cassandra/index/sasi/memory/KeyRangeIterator.java	2016-10-17 13:43:49.834944194 +0200
@@ -29,7 +29,7 @@
 import org.apache.cassandra.index.sasi.utils.CombinedValue;
 import org.apache.cassandra.index.sasi.utils.RangeIterator;
 
-import com.carrotsearch.hppc.LongOpenHashSet;
+import com.carrotsearch.hppc.LongHashSet;
 import com.carrotsearch.hppc.LongSet;
 import com.google.common.collect.PeekingIterator;
 
@@ -95,7 +95,7 @@
 
         public LongSet getOffsets()
         {
-            LongSet offsets = new LongOpenHashSet(4);
+            LongSet offsets = new LongHashSet(4);
             for (DecoratedKey key : keys)
                 offsets.add((long) key.getToken().getTokenValue());
 
@@ -126,4 +126,4 @@
             return keys.iterator();
         }
     }
-}
\ No newline at end of file
+}
diff -ur cassandra-cassandra-3.9/test/unit/org/apache/cassandra/index/sasi/disk/TokenTreeTest.java cassandra-cassandra-3.9new/test/unit/org/apache/cassandra/index/sasi/disk/TokenTreeTest.java
--- cassandra-cassandra-3.9/test/unit/org/apache/cassandra/index/sasi/disk/TokenTreeTest.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/test/unit/org/apache/cassandra/index/sasi/disk/TokenTreeTest.java	2016-10-17 13:43:49.836944211 +0200
@@ -43,7 +43,7 @@
 import junit.framework.Assert;
 import org.junit.Test;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
-import com.carrotsearch.hppc.LongOpenHashSet;
+import com.carrotsearch.hppc.LongHashSet;
 import com.carrotsearch.hppc.LongSet;
 import com.carrotsearch.hppc.cursors.LongCursor;
 import com.google.common.base.Function;
@@ -52,12 +52,12 @@
 {
     private static final Function<Long, DecoratedKey> KEY_CONVERTER = new KeyConverter();
 
-    static LongSet singleOffset = new LongOpenHashSet() {{ add(1); }};
-    static LongSet bigSingleOffset = new LongOpenHashSet() {{ add(2147521562L); }};
-    static LongSet shortPackableCollision = new LongOpenHashSet() {{ add(2L); add(3L); }}; // can pack two shorts
-    static LongSet intPackableCollision = new LongOpenHashSet() {{ add(6L); add(((long) Short.MAX_VALUE) + 1); }}; // can pack int & short
-    static LongSet multiCollision =  new LongOpenHashSet() {{ add(3L); add(4L); add(5L); }}; // can't pack
-    static LongSet unpackableCollision = new LongOpenHashSet() {{ add(((long) Short.MAX_VALUE) + 1); add(((long) Short.MAX_VALUE) + 2); }}; // can't pack
+    static LongSet singleOffset = new LongHashSet() {{ add(1); }};
+    static LongSet bigSingleOffset = new LongHashSet() {{ add(2147521562L); }};
+    static LongSet shortPackableCollision = new LongHashSet() {{ add(2L); add(3L); }}; // can pack two shorts
+    static LongSet intPackableCollision = new LongHashSet() {{ add(6L); add(((long) Short.MAX_VALUE) + 1); }}; // can pack int & short
+    static LongSet multiCollision =  new LongHashSet() {{ add(3L); add(4L); add(5L); }}; // can't pack
+    static LongSet unpackableCollision = new LongHashSet() {{ add(((long) Short.MAX_VALUE) + 1); add(((long) Short.MAX_VALUE) + 2); }}; // can't pack
 
     final static SortedMap<Long, LongSet> simpleTokenMap = new TreeMap<Long, LongSet>()
     {{
@@ -591,7 +591,7 @@
 
     private static LongSet convert(long... values)
     {
-        LongSet result = new LongOpenHashSet(values.length);
+        LongSet result = new LongHashSet(values.length);
         for (long v : values)
             result.add(v);
 
@@ -622,7 +622,7 @@
         {{
                 for (long i = minToken; i <= maxToken; i++)
                 {
-                    LongSet offsetSet = new LongOpenHashSet();
+                    LongSet offsetSet = new LongHashSet();
                     offsetSet.add(i);
                     put(i, offsetSet);
                 }
diff -ur cassandra-cassandra-3.9/test/unit/org/apache/cassandra/index/sasi/utils/LongIterator.java cassandra-cassandra-3.9new/test/unit/org/apache/cassandra/index/sasi/utils/LongIterator.java
--- cassandra-cassandra-3.9/test/unit/org/apache/cassandra/index/sasi/utils/LongIterator.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/test/unit/org/apache/cassandra/index/sasi/utils/LongIterator.java	2016-10-17 13:43:49.836944211 +0200
@@ -23,7 +23,7 @@
 import java.util.Iterator;
 import java.util.List;
 
-import com.carrotsearch.hppc.LongOpenHashSet;
+import com.carrotsearch.hppc.LongHashSet;
 import com.carrotsearch.hppc.LongSet;
 import org.apache.cassandra.db.DecoratedKey;
 import org.apache.cassandra.index.sasi.disk.Token;
@@ -84,7 +84,7 @@
         @Override
         public LongSet getOffsets()
         {
-            return new LongOpenHashSet(4);
+            return new LongHashSet(4);
         }
 
         @Override
diff -ur cassandra-cassandra-3.9/test/unit/org/apache/cassandra/index/sasi/utils/RangeIntersectionIteratorTest.java cassandra-cassandra-3.9new/test/unit/org/apache/cassandra/index/sasi/utils/RangeIntersectionIteratorTest.java
--- cassandra-cassandra-3.9/test/unit/org/apache/cassandra/index/sasi/utils/RangeIntersectionIteratorTest.java	2016-09-26 16:02:27.000000000 +0200
+++ cassandra-cassandra-3.9new/test/unit/org/apache/cassandra/index/sasi/utils/RangeIntersectionIteratorTest.java	2016-10-17 13:43:49.837944219 +0200
@@ -27,7 +27,7 @@
 import org.apache.cassandra.index.sasi.utils.RangeIntersectionIterator.BounceIntersectionIterator;
 import org.apache.cassandra.io.util.FileUtils;
 
-import com.carrotsearch.hppc.LongOpenHashSet;
+import com.carrotsearch.hppc.LongHashSet;
 import com.carrotsearch.hppc.LongSet;
 
 import org.junit.Assert;
@@ -311,7 +311,7 @@
             for (int i = 0; i < ranges.length; i++)
             {
                 int rangeSize = random.nextInt(16, 512);
-                LongSet range = new LongOpenHashSet(rangeSize);
+                LongSet range = new LongHashSet(rangeSize);
 
                 for (int j = 0; j < rangeSize; j++)
                     range.add(random.nextLong(0, 100));
